code/src/com/sixsq/nuvla/server/resources/deployment_set.clj

```diff
@@ -182,7 +182,7 @@ These resources represent a deployment set that regroups deployments.
 
 (defn create-app-set
   [{:keys [modules] :as resource} request]
-  (if (seq modules)
+  (if (some? modules)
```

Comment:
why do you change seq by some?
Because in case of empty list in modules it behaves not exactly the same. Didn't check more deeply than that.
``` 
(some? [])  ;; true
(seq [])    ;; nil
```

Reply:
Because I wanted the app-set to be created even if no apps are added to it.